sct.py

```diff
@@ -987,6 +987,15 @@ def unit_tests(test):
 @click.option("-t", "--test", required=False, default="",
               help="Run specific test file from unit-tests directory")
 def integration_tests(test):
+    get_test_config().logdir()
+    add_file_logger()
+    local_cluster = mini_k8s.LocalKindCluster(
```

Comment:
why do we need it?

Reply:
since we are taking a bare SCT runner, some of the system configuration that is done for local kind, we need it as well for the integration tests, yes I probability could add a comment here stating why we are doing this.

Reply:
Added a comment, saying exactly that